mock/py/mockbuild/config.py

```diff
@@ -356,6 +363,8 @@ def set_config_opts_per_cmdline(config_opts, options, args):
         config_opts['rpmbuild_arch'] = config_opts['target_arch']
     if options.forcearch:
         config_opts['forcearch'] = options.forcearch
+    # forcearch can be set from other places too
+    multiply_platform_multiplier(config_opts)
```

Comment:
Can we only call this in `check_arch_combination`?

Reply:
I can, but then the multiplier and forcearch option will be "out of sync" for most of `set_config_opts_per_cmdline()` and for several lines in `mock.py`.

Reply:
Hmm, ok ... I won't object too much honestly :-) it just looks too complicated to me, while we could just live with something like
https://github.com/praiskup/mock/commit/4f638bedadef476dd2bc66933cb6274ae4e2e439